monitor/dbt_project/macros/get_new_model_alerts.sql

```diff
@@ -9,6 +9,15 @@
     {% set new_alerts = [] %}
     {% for model_result_alert_dict in model_result_alert_dicts %}
         {% set status = elementary.insensitive_get_dict_value(model_result_alert_dict, 'status') | lower %}
+        {% set meta = fromjson(elementary.insensitive_get_dict_value(model_result_alert_dict, 'meta', '{}')) %}
+        {% set slack_channel = elementary.insensitive_get_dict_value(meta, 'channel') %}
+        {% set direct_subscribers = elementary.insensitive_get_dict_value(meta, 'subscribers', []) %}
+        {% set subscribers = [] %}
+        {% if direct_subscribers is string %}
```

Comment:
same here, isn't it safer to do in the python?

Reply:
I am not sure, what scenario is worrying you?